sdk/tests/deqp/functional/gles3/es3fNegativeBufferApiTests.js

```diff
@@ -92,9 +92,13 @@ goog.scope(function() {
                 gl.bufferSubData(gl.ARRAY_BUFFER, 15, data);
                 this.expectError(gl.INVALID_VALUE);
 
-                bufferedLogToConsole('gl.INVALID_VALUE is generated if data is null.');
-                gl.bufferSubData(gl.ARRAY_BUFFER, 0, null);
-                this.expectError(gl.INVALID_VALUE);
+                bufferedLogToConsole('An exception is thrown if data is null.');
+                try {
+                    gl.bufferSubData(gl.ARRAY_BUFFER, 0, null);
```

Comment:
Wish we could find a more terse way to express this. Would be nice if shouldThrow() in js-test-pre.js could handle the case where the argument was a function rather than a string that needed to be eval'd. Then a little anonymous function could be written:

  shouldThrow(function() { gl.bufferSubData(gl.ARRAY_BUFFER, 0, null); })

Not sure how "this" could be accessed from that inner function, but I don't think the expectError(gl.NO_ERROR) is strictly needed after the exception is thrown. Or that could be done after the shouldThrow statement.